packages/gatsby/src/redux/actions/restricted.ts

```diff
@@ -419,6 +421,32 @@ export const actions = {
         })
       }
     },
+
+  /**
+   * Enables/Disables a feature in Gatsby, this is helpful when your site is backed by a service like Gatsby Cloud.
+   *
+   * @availableIn [onPreInit]
+   *
+   * @param {string} name The feature name you want to toggle on and off
+   * @param {boolean} value The value you want it to set
+   * @example
+   * // When you want to enable the custom /_gatsby/file & /_gatsby/image endpoints
+   * exports.onPreInit = ({ actions }) => {
+   *   actions.toggleFeature('imageService', true)
+   * }
+   */
+  toggleFeature: (
```

Comment:
I get a bit confused with `toggle` sometimes, since `toggle` usually means `someBoolean = !someBoolean`. What are your thoughts on `setFeatureEnabled` to match the phrasing of `isFeatureEnabled`?

Reply:
I don't feel strongly about this so i'm fine with it 😄